babel.config.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
           "@theme": "./theme",
           "@design-system": "./design-system",
           "@navigation": "./navigation",
+          "@features": "./features",
```

Comment:
We may want to get more specific with these
like:
@search : "./features/search"

But for now totally fine